src/ophyd_async/core/signal_backend.py

```diff
@@ -35,6 +35,10 @@ async def get_reading(self) -> Reading:
     async def get_value(self) -> T:
         """The current value"""
```

Comment:
In line with making setpoint and readback slightly less confusing, could we replace this with:

```suggestion
    async def get_value(self) -> T:
        """The readback value of the signal."""
```